### DISCUSSION TITLE
Display data from Array in Binding

### DISCUSSION THREAD

**Discussion by sangroxx** [Budibase/budibase#4018] [Help] [answered]
Hi, I am trying to get set a paragraph text withing a repeater container.

This is paragraph text is using binding to display the last person who updated the row.

Using the handlebar is giving me an array of data -
```
Input - 
{{ Repeater.Orders.Updated By }}

Output -
[{"_id":"ro_ta_users_us_94f42ff28ad346df8e27940cdd6e6ae8","primaryDisplay":"email@domain.dev"}]

```

I want to split this to only display the name of the person who updated the data (the part after "primaryDisplay" in the above code)

Tried, split, after and more helpers but nothing seemed to work.

**Comment by mjashanks** [comment, marked as answer]
Hi @sangroxx - try `{{ Repeater.Orders.Updated By.1.primaryDisplay }}`

**Reply by sangroxx** [reply]
I tried that, is not working.. there is no output. just blank.  (Yes I did click on save after making the change)

(Updated the attached image with correct input)

![image](https://user-images.githubusercontent.com/10852022/149337840-9a22965f-6e33-4773-bdc7-6a85e0a4d655.png)

**Reply by mjashanks** [reply]
apologies - it's `{{ Repeater.Orders.Updated By.0.primaryDisplay }}`  (`0` not `1`) - just tested this on my end and it working ok.

By default, this will display the email of the person. If you want to display their name:
1. Go to your Users table
2. Edit the `First Name` column
3. Set "Use as display"

If you need the Full Name, you can create a new formula column, and use this as the primary display.

**Reply by sangroxx** [reply]
Hi, yes that worked! Thank you so much.

However, there is just no way to edit the First Name column in the Users table. Did the Full Name work around though. Worked like a charm.

**Comment by sangroxx** [comment]
Hi there again. This seems to have stopped working. And when I check the last update by column in the Data.. it is all empty.

It seems like budibase not tracking the users who have made change to the row.

**Reply by mjashanks** [reply]
Hi @sangroxx - I have just check this end-to-end, with different users and it seems to be working ok for me. My Steps
1. Setup a binding on the edit screen, as above
2. Open app and update record.
3. Re-open record - Binding shows correctly
4. Now, log in with another user and repeat. Again, is working.

Are you running Budibase self-hosted? If so, have you updated recently? Do you take any different steps than to the above?

**Reply by sangroxx** [reply]
How do I update Budibase? The update button in the builder says I am on the latest version, however I just setup an instance on AWS and I guess that's the latest version, but I can't find the version number on it.

![image](https://user-images.githubusercontent.com/10852022/156885933-26d13682-f7db-4c79-a271-dbf37e486e5a.png)

The latest one is v1.0.8 I suppose? 

Sorry, absolute noob.

**Reply by mjashanks** [reply]
That's weird. Budibase uses watchtower to manage updates. It's possible that your watchtower docker container is not running - please check this.

You can still update manually, using `docker-compose`, or the budibase cli -`budi hosting --update`

**Reply by sangroxx** [reply]
Yup, watchtower isn't running.. don't know what is the issue. Looking into it.

However, having the same issue in the budibase account app - [https://dafter.budibase.app/builder/app/app_dev_dafter_1ed985543d764a54bc29e6b78a2db258/data/table/ta_dac5d07102b643878bf00a8311501f03](https://dafter.budibase.app/builder/app/app_dev_dafter_1ed985543d764a54bc29e6b78a2db258/data/table/ta_dac5d07102b643878bf00a8311501f03)
